Makefile

```diff
@@ -802,7 +801,7 @@ deploy/service: deploy/envoy deploy/route
 		-p STRIMZI_OLM_PACKAGE_NAME="${STRIMZI_OLM_PACKAGE_NAME}" \
 		-p KAS_FLEETSHARD_OLM_PACKAGE_NAME="${KAS_FLEETSHARD_OLM_PACKAGE_NAME}" \
 		-p CLUSTER_LIST="${CLUSTER_LIST}" \
-		-p SUPPORTED_CLOUD_PROVIDERS="${SUPPORTED_CLOUD_PROVIDERS}" \
+		-p SUPPORTED_CLOUD_PROVIDERS=${SUPPORTED_CLOUD_PROVIDERS} \
```

Comment:
This is in the second commit. It was causing some errors 
```
error: template name must be specified only once: default:
See 'oc process -h' for help and examples.
```
and I had to remove the quotes to make everything work again.